Remove.py

```diff
@@ -1,35 +1,48 @@
 import PrintList
 
 
-def remove_1(to_Do, removed):
+def remove_1(to_Do):
 
   print("""
   ------------------------------
   Remove an item
   ------------------------------
   """)
-  
+
+  # Print current list
   PrintList.print_list(to_Do)
 
-  X = input("[Enter X to exit this screen]")
+  #User presses X to exit screen
+  X = input("[Enter X to exit this screen]").upper()
 
-  item_to_remove = int(input("What item would you like to remove?")) - 1
+  if X == "X":
+    return
+  else:
+    pass
 
-  
-  # Calling the item by index and not by name
-  if item_to_remove <= len(to_Do):
+  for item_to_remove in range(3):
+    
+    item_to_remove = input("What item would you like to remove?").upper()
 
-    user_choice = to_Do[item_to_remove]
+    if item_to_remove == "X":
+      return
 
-    to_Do.remove(user_choice)
-    removed.append(user_choice)
+    try:
+      task = int(item_to_remove) - 1
+      if 0 <= task <= len(to_Do):
+        user_choice = to_Do[task]
+        to_Do.remove(user_choice)
+        PrintList.print_list(to_Do)
+        return
 
-  elif X == "X":
-    return
+      else:
+        print("Enter a number within range!")
```

Comment:
One style of coding (People have different styles, so I'm just presenting this info. Feel free to ignore it or to implement it.) would be to move the failure condition to the _top_ of the `if` check. 

So instead of
```
if condition met:
  Do
  All
  This
  Work
else:
  Print an error message
```
we'd instead do
```
if condition not met:
  Print an error message
else:
  Do
  All
  This
  Work
```
This style can make code easier to read, especially in deeply nested checks. For example, which of the following is easier to follow? ;)
```
if condition 1 met:
  Do something
  if condition 2 met:
    Do another thing
    if condition 3 met:
      Do yet another thing
    else:
      Print error message 3
  else:
    Print error message 2
else:
  Print error message 1
```
or
```
if condition 1 not met:
  Print error message 1
else:
  Do something 
  if condition 2 not met:
    Print error message 2
  else:
    Do another thing
    if condition 3 not met:
      Print error message 3
    else:
      Do yet another thing
```

Reply:
Okay, will see how to implement it! Thank you for this advice! It is really confusing to read the first option!

Reply:
I do have a question. In the if statement 0 <= task <=len(to_Do) i added 0 <= task < len(to_Do) instead and when i add a number outside of the index range it doesnt give me an out of index error it just print the input statement again. Why does it do that?

Reply:
We covered this during our meeting today. Please play with moving the "- 1" on the user input from outside the if statement to inside the if statement and seeing how that breaks the system and what changes need to be made to fix the system. Just to get comfortable with off by one errors.